sdccc/src/test/java/com/draeger/medical/sdccc/manipulation/precondition/BufferedObservingPreconditionTest.kt

```diff
@@ -130,9 +132,20 @@ internal class BufferedObservingPreconditionTest {
 
         exampleObserving.observeChange(mock<MdibChange.Metric>())
 
-        verify(testRunObserver, times(1)).invalidateTestRun(anyString(), any())
+        exampleObserving
+            .processingThread
+            .join(TIME_TO_WAIT_FOR_CHANGE.inWholeMilliseconds)
+
+        verify(
+            testRunObserver,
+            times(1)
+        ).invalidateTestRun(anyString(), any())
 
         // passing more changes is still possible
```

Comment:
Actually wrong now and previously only sometimes correct.

It will not add anything and instead call clear and return.